### PR TITLE
add

### PR DESCRIPTION
**Pull request by vkhmara**
I deleted all images but 2 "like" images. So because of that I have changed .java files.

### CODE REVIEW

**Comment by skateray17** [comment]
I can't test it locally. Trying to run, I'm getting that error. Tried to uncomment call of `fullingStorage`, but it also fails with error `Uncaught ReferenceError: Cannot access 'postsArray' before initialization`
image

**Comment by vkhmara** [comment]
I had deleted my first array of posts. I fixed it

File: task8/ServletsOnMaven/src/main/webapp/resources/js/View.js [hunk 1]
```diff
@@ -0,0 +1,1001 @@
...
+		}
+	});
+	localStorage.setItem("Users", JSON.stringify(Array.from(users)));
+	localStorage.setItem("Current user", JSON.stringify(null));
+	localStorage.setItem("Count of posts in feed", "10");
+	localStorage.setItem("IDs of current feed", JSON.stringify(idsOfPosts));
+})();*/
+
+function loadPostFromStorage(id) {
+	let post = JSON.parse(localStorage.getItem("Post with id=" + id));
```
Comment (skateray17) [inline task8/ServletsOnMaven/src/main/webapp/resources/js/View.js:267]: Why you stored each post on it's own key? You could store just array of posts

File: task8/ServletsOnMaven/src/main/webapp/resources/js/View.js [hunk 2]
```diff
@@ -0,0 +1,1001 @@
...
+	return new Post(post.id, post.author, new Date(post.createdAt), post.description, post.hashTags, post.likes)
+}
+
+(function loadInfoFromStorage() {
+	idsOfPosts = JSON.parse(localStorage.getItem("IDs of posts"));
+	idsOfCurrentFeed = JSON.parse(localStorage.getItem("IDs of current feed"));
```
Comment (skateray17) [inline task8/ServletsOnMaven/src/main/webapp/resources/js/View.js:273]: Why should it be stored in LS?

File: task8/ServletsOnMaven/src/main/webapp/resources/js/View.js [hunk 3]
```diff
@@ -0,0 +1,1001 @@
...
+	idsOfPosts.forEach(id => postsArray.push(loadPostFromStorage(id)));
+	JSON.parse(localStorage.getItem("Users")).forEach(item => users.set(item[0], new User(item[1].username, item[1].password)));
```
Comment (skateray17) [inline task8/ServletsOnMaven/src/main/webapp/resources/js/View.js:275]: Why do you duplicate username?

File: task8/ServletsOnMaven/src/main/webapp/resources/js/View.js [hunk 4]
```diff
@@ -0,0 +1,1001 @@
...
+let allPosts = new PostContainer(postsArray);
+
+class ViewPosts {
+
+	constructor() {
+		this.posts = new PostContainer();
+		idsOfCurrentFeed.forEach(id => {
+			this.posts.addPost(allPosts.getPost(id));
+		});
+		this.count = parseInt(JSON.parse(localStorage.getItem("Count of posts in feed")));
```
Comment (skateray17) [inline task8/ServletsOnMaven/src/main/webapp/resources/js/View.js:287]: Again, why it should be stored in LS? After reload, again 10 posts should be shown, but not the same amount as before reload

File: task8/ServletsOnMaven/src/main/webapp/resources/js/View.js [hunk 5]
```diff
@@ -0,0 +1,1001 @@
...
+<div class="header_of_window">Error</div>
+<div class="info_error_window">
+
+</div>
+<div class="buttons_field">
+<button class="main_button_of_error">
+Main
+</button>
+<button class="back_button">
+Back
```
Comment (skateray17) [inline task8/ServletsOnMaven/src/main/webapp/resources/js/View.js:562]: It does just nothing? 🙁

**Review by skateray17 — APPROVED** [review]

**Pull request merged**